reversestring-SLG/style.css

```
@@ -0,0 +1,260 @@
+  --bg:         #0d0f14;
+  --surface:    #13161e;
+  --border:     rgba(255,255,255,0.08);
+  --border-hi:  rgba(255,255,255,0.18);
+  --accent:     #f6c26b;
+  --accent-dim: rgba(246,194,107,0.12);
+  --text:       #e8e4dc;
+  --muted:      #6b6e7a;
+  --green:      #6bdfa6;
+  --radius:     18px;
+  --mono:       'DM Mono', monospace;
+  --serif:      'Playfair Display', Georgia, serif;
```

Stylelint (17.4.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)